std_msgs/Header header

int32 id       # ID of anchor
float64 range  # distance to anchor
